specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2019-04-01/containerregistry_build.json

```diff
@@ -0,0 +1,2460 @@
+{
```

Comment:
Please, create two commits. The first ne should contain a copy of the old `containerregistry_build.json`. The second commit should contain changes in the new API version. We will review the second commit.